app/models/package_manager/base.rb

```diff
@@ -180,7 +180,7 @@ def self.add_version(db_project, version_hash)
 
     def self.deprecate_versions(db_project, version_hash)
       case db_project.platform.downcase
-      when "rubygems" # yanked gems will be omitted from project JSON versions
+      when "rubygems", "npm" # yanked gems will be omitted from project JSON versions
```

Comment:
Were you able to confirm this behavior on a package with a removed release?

Reply:
I did not do any tangible manual testing of this beyond ensuring the specs behaved as intended. I feel the specs are sufficient, and given the research @johnbintz-tidelift has previously done to identify the need for this ticket, I think this covers what we require, and we can confirm it works after implementation. But if you have suggestions for something you'd like me to check I can investigate that.